### ISSUE TITLE
Project 1 Due Date (also Dropbox)

### ISSUE THREAD

**Issue by WesleyTo** [stefanbund/cis155#13]
I'm still a little confused by the due dates. What's been posted in the forums by @kayvee and @stefanbund, and what's in the milestone guide/project roadmap don't seem to quite add up for me. Could someone please clarify?

Milestone Guide

```
Project 1 due March 5th
Milestone 1
Weeks go Sunday to Sunday
1. 1/ 31 - 2/ 6th.  Lesson 1, 2, 3, 4
2. 2/ 6  - 13th. Lessons 5, 6, 7, 8
Milestone 2
1. 2/ 20 - 2/ 27 Lessons 9, 10, 11, 12
2. 2/ 27 - 3/ 5 Lesson 13, 14, 15, 16
```

The project due date (March 5) is the same as the milestone 2 due date, but based on the project roadmap and the content in videos 9-16, milestone 2 only covers through the end of phase 1. Videos 17-23 (which aren't listed at all in the Milestone guide) only seem to cover phase 2. Phase 3 doesn't seem to be covered at all as of yet.

Also, @kayvee mentions and @stefanbund seems to have confirmed in the week 3 forum that phase 2 would be due 5 weeks after phase 1. I thought that projects were due every 5 weeks, not phases (hence the April 9th due date for project 2).

So is the project due date (March 5) correct? Are we supposed to have all 3 phases done already? If so, when will the dropbox be opened for it?

**Comment by kayvee** [comment]
@WesleyTo 

I can't comment on the due dates or the videos, which I do think don't quite add up to the project all the time. But I think you have pointed out one issue with the terminology in the guides:

We have **one** _project_. This is detailed in the roadmap as a stock market project.
It has **three** _phases_. These are the separate portions in the project roadmap.
Each phase has **two** _milestones_. These are just checkups to see how far you have progressed to keep you on track with feedback.

To write out the dates for all of them the way I understood it (with numbers indicating weeks, Monday to Sunday):
1. [ 1/30-2/7 ] - 
2. [ 2/8/-2/14 ] - milestone 1 (phase 1)
3. [ 2/15-2/21 ] - 
4. [ 2/22-2/28 ] - milestone 2 (phase 1)
5. [ 2/29-3/6 ] - phase 1 _due on 3/5_
6.  [ 3/7-3/13 ] - **[CURRENT WEEK]**
7. [ 3/14-3/20 ] - milestone 3 (phase 2)
8. [ 3/21-3/27 ] - 
9. [ 3/28-4/3 ] - milestone 4 (phase 2) **[ALSO SPRING RECESS]**
10. [ 4/4-4/10 ] - phase 2 _due on 4/9_
11. [ 4/11-4/17 ] - 
12. [ 4/18-4/24 ] - 
13. [ 4/25-5/1 ] - milestone 5 (phase 3)
14. [ 5/2-5/8 ] - 
15.  [ 5/9-5/15 ] - milestone 6 (phase 3)
16. [ 5/16-5/22 ] - phase 3 _due on 5/21_
17. [ 5/23-5/29] - last week, nothing due, perhaps just the project as a whole?

**Comment by kayvee** [comment]
Now for the video content. I have made a single playlist with all of our videos thus far:

https://www.youtube.com/playlist?list=PLT7-ZAJRz9NQUe_duz1KbmpBc4X6YhMJl

I think it correlates as:
- [ Milestone 1 ] - 1, 2, 3, 4, 5, 6, 7, 8 part 1, 8 part 2
- [ Milestone 2 ] - 9, 10, 11, 12, 13, 14, 15, 16 part 1, 16 part 2
- [ Phase 1 ] - review 1-16
- [ Milestone 3 ] - phase 2 introduction, 17, 18, 19, 20, 21, 22, 23
- [ Milestone 4 ] - tba
- [ Phase 2 ] - review 17-tba
- [ Milestone 5 ] - tba
- [ Milestone 6 ] - tba
- [ Phase 3 ] - tba
- [ Project ] - tba

**Comment by kayvee** [comment]
I think we are waiting for the following now as well:
- dropbox discussion post 5
- dropbox phase 1
- discussion post question week 6

**Comment by stefanbund** [comment]
OK, I will get these online

From Stefan

**Comment by Irfan009** [comment]
@kayvee 
Thank you.

**Comment by WesleyTo** [comment]
@kayvee Thanks!.
@stefanbund, I also saw the revised _milestone guide_ you just posted.

So @kayvee is correct about the project structure in the second post of this thread.

Then in the _milestone guide_ (both the old and revised ones), instead of:

> CIS 155 project roadmap: provides guidelines on what to deliver for the 3 projects
> ...
> Projects are broken into three phases. As you can read in the Syllabus, the first and
> second projects are worth 13 points. The final project is worth 14 points. The two
> milestones are due per project, for a total of 6 milestones. Each milestone is worth 5%
> points, or 30% of the final grade.

It should read:

> CIS 155 project roadmap: provides guidelines on what to deliver for the 3 **_phases_**
> ...
> Projects are broken into three phases. As you can read in the Syllabus, the first and
> second **_phases_** are worth 13 points. The final **_phase_** is worth 14 points. **_There are_** two milestones due per **_phase_**, for a total of 6 milestones. Each milestone is worth 5%
> points, or 30% of the final grade.

**Comment by stefanbund** [comment]
We will have two phases instead of three. This will help folks to catch up. I did not include the point spread, but it will cover the same amount of points, but spread across two phases, not three

From Stefan

**Comment by WesleyTo** [comment]
@stefanbund Gotcha! Thanks!

**Comment by kayvee** [comment]
@stefanbund 

Thank you for clarifying everything. Will you still provide videos for phase 3/more advanced materials, however? Or does the course 'end' after video 25? i.e., if we are already done with Phase 1 and we complete Phase 2, are we 'done' with the course?

**Comment by stefanbund** [comment]
Technically yes, but discussions will also be required

From Stefan
